/dr_spaam_ros:
  ros__parameters:
  
    weight_file: "ckpts/ckpt_jrdb_ann_drow3_e40.pth"
    detector_model: "DROW3"  # DROW3 or DR-SPAAM
    # weight_file: "ckpts/ckpt_jrdb_ann_dr_spaam_e20.pth"
    # detector_model: "DR-SPAAM"  # DROW3 or DR-SPAAM
    conf_thresh: 0.5
    stride: 1  # use this to skip laser points
    panoramic_scan: True  # Set to true if the scan covers 360 degree

    publisher:
        detections:
            topic: /dr_spaam_detections
            queue_size: 1
            latch: false

        rviz:
            topic: /dr_spaam_rviz
            queue_size: 1
            latch: false

    subscriber:
        scan:
            topic: /philbart/scan
            queue_size: 1
